book/src/prefabs/prefabs_technical_explanation.md

```diff
@@ -0,0 +1,323 @@
+# Prefabs Technical Explanation
+
+A `Prefab` in Amethyst is at the core a simple list of future entities, where each entry in
+the list consists of two pieces of optional data:
+
+* a parent index that refers to a different entry in the list
+* a data collection implementing the trait `PrefabData`
+
+To instantiate a `Prefab`, we put a `Handle<Prefab<T>>` on an `Entity`. The `Entity` we put
```

Comment:
This handle? https://www.amethyst.rs/doc/master/doc/amethyst_assets/struct.Handle.html
Maybe specify which handle is the right? (searching "Handle" in the doc gives 179 results)